embed/config.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/coreos/etcd/pkg/transport"
 	"github.com/coreos/etcd/pkg/types"
 	"github.com/ghodss/yaml"
+	"google.golang.org/grpc"
```

Comment:
can you separate the internal and external dependencies?

```
"github.com/coreos/etcd/pkg/transport"
  	"github.com/coreos/etcd/pkg/types"

  	"github.com/ghodss/yaml"
        "google.golang.org/grpc"
```

I know the problem is not introduced by you. but since you are already on it, please help to fix. Thanks!